packages/adblocker/package.json

```diff
@@ -58,7 +58,8 @@
     "codebook-cosmetic-selector": "tsx ./tools/generate_compression_codebooks.ts -- cosmetic-selector",
     "codebook-raw-network": "tsx ./tools/generate_compression_codebooks.ts -- raw-network",
     "codebook-raw-cosmetic": "tsx ./tools/generate_compression_codebooks.ts -- raw-cosmetic",
-    "generate-codebooks": "concurrently -n build: \"yarn:codebook-*\" && yarn bump-internal-engine-version"
+    "generate-codebooks": "concurrently -n build: \"yarn:codebook-*\" && yarn bump-internal-engine-version",
+    "test-samples-compression": "tsx ./tools/test-samples-compression.ts"
```

Comment:
not sure if this is needed, I would prefer to remove it

Reply:
The internal tooling was replaced with brotli CLI in the following commit: [`7aeb03b` (#3897)](https://github.com/ghostery/adblocker/pull/3897/commits/7aeb03bdb3959fed699da867e1c7d70a6da26120)